protostar/configuration_file/configuration_file_v2_test.py

```diff
@@ -16,11 +16,20 @@
     ConfigurationFileContentConfigurator,
     ContractNameNotFoundException,
 )
-from .configuration_file_v1 import ConfigurationFileV1, ConfigurationFileV1Model
+from .configuration_file_v1 import (
+    CommandNamesProvider,
+    ConfigurationFileV1,
+    ConfigurationFileV1Model,
+)
 from .configuration_file_v2 import ConfigurationFileV2, ConfigurationFileV2Model
 from .configuration_legacy_toml_interpreter import ConfigurationLegacyTOMLInterpreter
 
 
+class CommandNamesProviderDouble(CommandNamesProvider):
```

Comment:
What does the name mean?

Reply:
https://martinfowler.com/bliki/TestDouble.html

Reply:
I would name it `Stub` instead, name `Dobule` does not seem very popular one

Reply:
I thought it has some property of doing something twice/ returning two commands

Reply:
The name is also used in NestJS docs.
https://docs.nestjs.com/fundamentals/testing#unit-testing

Anyway, I can rename it.

Reply:
It may be but I believe stub is more popular one and less misleading